### ISSUE TITLE
Release manager required

### ISSUE THREAD

**Issue by ghost** [mpv-player/mpv#872]
We need someone who cares about releases. This involves the following tasks
- deciding when a release should be made, and based on which git commit
- writing release notes, giving an overview over what changed (features, bug fixes)
- publishing the release
- maintaining the release (cherry picking commits from master, and making minor releases based on the release branch)
- git skills and (rudimentary) C skills are needed

You also have the freedom to change the current release policy.

You will get git access if you don't already, so that you can push the release branch, release tags, and use the github release system.

Basically I don't care about releases at all. I'll be maintaining the 0.3.x branch for a while, but there will be no 0.4.0 without a new release manager.
